src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import classNames from 'classnames';
+import { Error } from '../../types/Error';
+
+type Props = {
+  error: Error | null;
+  setError: (value: null) => void;
+  setIsLoading: (value: boolean) => void;
+};
+
+export const ErrorNotification: React.FC<Props> = ({
+  error,
+  setError,
+  setIsLoading,
+}) => {
+  if (error) {
+    setTimeout(() => {
+      setError(null);
+      setIsLoading(false);
```

Comment:
but why do you need additional setIsLoading inside the `ErrorNotification` component?